senza/templates/__init__.py

```diff
@@ -24,6 +24,7 @@ def get_templates() -> dict:
             continue
         else:
             # make sure the entry point resolves to a module with the essential interface functions
-            if isinstance(module, ModuleType) and has_functions(module, ('gather_user_variables', 'generate_definition')):
+            if isinstance(module, ModuleType) and\
```

Comment:
Can you use `()` instead of `\`, as recommended by [PEP8](https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator)?